app/apollo/resolvers/organization.js

```diff
@@ -26,6 +31,81 @@ const organizationResolvers = {
       return models.User.getOrgs(context);
     },
   },
+  Mutation:{
+    createOrgEncKey: async (parent, { orgId }, context)=>{
+      const { models, me, req_id, logger } = context;
+      const queryName = 'createOrgEncryptionKey';
+      logger.debug({req_id, user: whoIs(me), orgId }, `${queryName} enter`);
+      await validAuth(me, orgId, ACTIONS.MANAGE, TYPES.ORGANIZATION, queryName, context);
+
+      const org = await models.Organization.findOne({ _id: orgId }).lean();
+      if(!org || !org._id){
+        throw new RazeeValidationError(context.req.t('org id was not found'), context);
+      }
+      const liveKeys = _.filter(org.encKeys||[], (encKey)=>{
+        return !encKey.deleted;
+      });
+      if(liveKeys.length >= 10){
+        throw new BasicRazeeError(context.req.t('This org has too many encryption keys. Remove some before adding any new ones'), context);
+      }
+
+      const encKey = await genKey();
```

Comment:
```suggestion
      const encKey = genKey();
```
genKey doesn't appear to return a promise